### ISSUE TITLE
Distinct classes for pagination links

### ISSUE THREAD

**Issue by lorenzulrich** [patriceckhart/NeosRulez.FusionLoopPagination#1]
It would be nice to have distinct classes for pagination links:

* first
* previous
* next
* last

Since the package is built with AFX, currently the whole `NeosRulez.Prototypes:Pagination` must be overridden to achieve this.

**Comment by patriceckhart** [comment]
@lorenzulrich Good idea. I'll be adding that soon. But you are also welcome to make a PR.
